eloquent/distribution.yaml

```diff
@@ -2226,7 +2226,7 @@ repositories:
     doc:
       type: git
       url: https://gitlab.com/micro-ROS/ros_tracing/ros2_tracing.git
-      version: 0.2.10
+      version: eloquent
```

Comment:
This is fine, but I'll also point out that you should update your release tracks.yaml [here](https://gitlab.com/micro-ROS/ros_tracing/ros2_tracing-release/-/blob/master/tracks.yaml#L39) to point to `eloquent` as well.

Reply:
oh, right, thanks for pointing that out!